core/commands/partials/terraform_configurator.rb

```diff
@@ -26,16 +27,20 @@ def initialize(config, env, logger)
                         else
                           NetworkSettings.new
                         end
+    @threads_count = @env.threads_count
```

Comment:
The amount of threads must depend on the number of machines that we are bringing up. All costly operations are run on the remote machines and must not depend on the number of cores that are available on the host machine.